Resources/Private/Translations/en/Main.xlf

```diff
@@ -191,6 +191,15 @@
 			<trans-unit id="searchBoxLeftToType" xml:space="preserve">
 				<source>Please enter ###CHARACTERS### more character</source>
 			</trans-unit>
+            <trans-unit id="Containers.LeftSideBar.DocumentTree.heading" xml:space="preserve">
```

Comment:
I don't think we use labels from here, instead they need to be put into Neos.Neos, as that is translateable through crowdin.
Alternatively we could move all content module labels from Neos.Neos into Neos.Neos.Ui and setup the Crowdin integration.

Reply:
Okay, moved them out of the UI package ;-)

Reply:
Btw, if by any chance you (or somebody else reading this) wants to take care of sorting the Translations mess, that would be cool. Things to do:
1) Discuss with the team how we want to proceed: 
  a) delete everything from Neos.Neos.Ui and leave everything in Neos.Neos (easy, just `rm -rf Resources/Private/Translations`)
  b) move all content module labels to Neos.Neos.Ui and setup crowdin integration for Neos.Neos.Ui (hard)
2) do it